### PR TITLE
Fixes #202: disable deploy link if repository is not from github

### LINKED ISSUE #202: disable deploy link in email if repository is not from github
### This is a
- [x] Bug report
- [ ] Feature request

### Current behavior

deploy link is sent as email even though repository is not from GitHub
### Expected behavior

don't send deploy link if the repository is not on Github

### Steps to reproduce
1.
2.
3.

### PR DESCRIPTION
**Pull request by sch00lb0y**
## Description

I hava removed the deploy link if repository not on github
## Related Issue

clsoe #202 
## Motivation and Context

now webui can deploy to gh-pages, so if the repository not from github remove the link
## Screenshots (if appropriate):
email
## Types of changes

- [x] Bug fix 
- [ ] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

File: backend/mailer.js [hunk 1]
```diff
@@ -18,10 +18,16 @@ exports.sendEmail = function (data) {
 
   var previewURL = 'http://' + hostname + '/preview/' + data.email + '/' + data.uniqueId + '_preview';
   var downloadURL = 'http://' + hostname + '/download/' + data.email + '/' + data.uniqueId;
-  var deployURL = 'http://' + hostname + '/github?email=' + data.email + '&uniqueId=' + data.uniqueId + '&gitURL=' + data.gitUrl;
+  var deployURL
```
Comment (imujjwal96) [inline backend/mailer.js]: do not use `undefined` variables. Initialize it with an empty string. Also don't forget to add `;` at the end of a statement

File: backend/mailer.js [hunk 2]
```diff
@@ -18,10 +18,16 @@ exports.sendEmail = function (data) {
...
+  if (validation.isGithubHTTPS(data.gitUrl)) {
+    deployURL = 'http://' + hostname + '/github?email=' + data.email + '&uniqueId=' + data.uniqueId + '&gitURL=' + data.gitUrl;
+  }
 
   var textContent = 'Hey! Your documentation generated successfully. Preview it here: ' + previewURL +
-                    '. Download it here: ' + downloadURL + '. Deploy it here: ' + deployURL;
+                    '. Download it here: ' + downloadURL ;
+  if (deployURL != undefined) {
```
Comment (imujjwal96) [inline backend/mailer.js]: use `!==` in place of `!=`

File: backend/mailer.js (hunk 1 above)
Comment (sch00lb0y) [inline backend/mailer.js]: made changes

File: backend/mailer.js (hunk 2 above)
Comment (sch00lb0y) [inline backend/mailer.js]: made changes

**Pull request merged**